limited/keen coyote

```diff
@@ -0,0 +1,28 @@
+**Keen Coyote** | Werewolf Group | Limited
+__Basics__
+The Keen Coyote shares a secret channel with the other coyotes and can perceive players each night to determine whether they have a threatening presence.
+__Details__
+The Keen Coyote cannot see a player’s role, but they can check whether or not they are a threat. Lycans, and roles classified as killing roles have a threatening presence.
+The Keen Coyote’s inspection is an immediate ability.
+The Keen Coyote is affected by strong disguises and redirections.
+The Keen Coyote knows the identity of the other coyotes and shares a channel with them.
+The Keen Coyote is not a lycan, nor are they a member of the wolfpack.
```

Comment:
Can we merge some of these so we don't have 6 lines in a row that start with "The Keen Coyote"